app/src/main/java/com/example/tasky/agenda_details/presentation/AgendaDetailsRoot.kt

```diff
@@ -58,7 +65,10 @@ fun AgendaDetailsRoot(
     agendaDetailsViewModel: AgendaDetailsViewModel = hiltViewModel()
 ) {
     val viewState by agendaDetailsViewModel.viewState.collectAsStateWithLifecycle()
+    val photosUri by agendaDetailsViewModel.photosUri.collectAsStateWithLifecycle()
+    val photoSkipCount by agendaDetailsViewModel.photoSkipCount.collectAsStateWithLifecycle()
```

Comment:
Why did you decide to not make these part of the screen state?

Reply:
I am going to fix this, add variables to my viewState that has any value that can change and has an effect on the screen. follow MVI architectural pattern.